### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by ALewis1975** [jetelain/ArmaRealMap#178] [General]
Does anyone know what causes, and how to fix, these errors:

No weather defined in .Overcast
No lighting defined in .Lighting

Thanks in advance.

**Comment by Toxicice02340** [comment]
I have been getting a similar error about lighting when trying to use a custom asset configuration on my maps. If you are using one as well and keep loading up a map that is dark but can have units placed down in Eden Editor that function, we may have the same problem. I do not know what specifically causes this, but this issue seems to only arise for me when I try loading up and individually created and saved preset of my own that is not one of the 2 presets. 

What I did was select an area I wanted to see generated, choose centraleurope config, edit that without saving it as a custom configuration, then generated it as map and the lighting seems to work just fine. Again, I don't know if we are having the exact same problem as I am not getting a .Overcast error, but it might be worth a try. As for why this works, what the specific cause is, or how to fix it in any other way, I don't know I'm just a beginner fiddling around with map creation and this app.
